projects/json-c/project.yaml

```diff
@@ -0,0 +1,4 @@
+homepage: "https://json-c.github.io/json-c/"
+primary_contact: "<primary_contact_email>"
```

Comment:
who will be a primary contact, @hawicz ?

Reply:
You can use erh+git@nimenees.com